src/components/TopNav/index.tsx

```diff
@@ -171,12 +171,12 @@ const TopNav: FC<ComponentProps> = ({className}) => {
         <Link className={clsx('TopNav__right-item', 'TopNav__download-button')} tabIndex={-1} to="/download">
           <Button>Download</Button>
         </Link>
-        {/* <A
+        <A
           className={clsx('TopNav__right-item', 'TopNav__download-button')}
-          href={`https://github.com/login/oauth/authorize?client_id=${process.env.REACT_APP_GITHUB_CLIENT_ID}&scope=read%3auser%20user%3aemail%20`}
+          href={`${process.env.REACT_APP_BACKEND_API}`}
```

Comment:
This particular line will remain same I think. Because the user will route to the github login page where the sign in will happen and the user will get redirected to a callback url having `code` in query params which will be sent to `REACT_APP_BACKEND_API` so that further backend can use that code to retrieve access token. 
I hope I am right :p

Reply:
Also let's keep this commented until the whole flow is done.

Reply:
not quite right, we are doing auth on a backend side, so, the flow is to go to our URL, that will redirect to a proper github URI

Reply:
@vosi So if understood this right, this is the flow:-
1) User clicks on login and request for login goes to backend. 
2) Backend redirects the user to the Github sign in URL, if user is not already signed in.
3) User sign's in and is redirected to our website's callback URL with `code` which we send back to backend.